guides/posts/pratham/introduction-to-api-testing/post.md

```diff
@@ -1 +1,59 @@
+---
+title: "Introduction to API Testing"
+description: "API testing is performed to test whether a particular API meets pre-defined parameters or not."
+slug: "2021-09-28-post-3"
+authors: ["pratham"]
```

Comment:
Authors and tags should be in [this format](https://raw.githubusercontent.com/RapidAPI/DevRel-Stack-Data/dev/guides/posts/pratham/create-a-text-sentiment-analysis-app/index.md).